[CS2113T-T09-4] PulsePilot

JustinSoh opened the pull request:
PulsePilot is a desktop app for tracking health-related information, optimised for users via a Command Line Interface (CLI). If one can type fast, you can key in and track health-related information faster than traditional GUI applications installed on your phone or computer.

LuoYu-uwu commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,662 @@
+5. `GymSet` stores the weight and repetitions for a particular set.
+6. `WorkoutList` is a class that stores an array list different `Workout` objects using ArrayList.
+
+![Workout Package Class Diagram](img/workout_class_diagram.png)
```
Maybe consider including labels and multiplicities for easier understanding.

Leong-ZM replied:
classes should be represented by text such as {abstract} instead of circled letters in the diagram

LuoYu-uwu commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,662 @@
+
+The main entry point of the application is the `Handler` class, used to handle user input. When the user starts the bot, the steps are as follows:
+
+1. `initialiseBot()` is called, and it prompts the user to enter their name, which is used throughout the bot.
+2. `initialiseScanner()` is used to create the singleton `Scanner` instance to read user input.
+3. When the user enters input, it is passed to `processInput()`, which determines the type of command inputted.
+4. Once the correct input has been determined, it is passed to the relevant handle function. For example, a `workout` command is passed to `handleWorkout()`.
```
Is it better to include a sequence diagram?

LuoYu-uwu commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,662 @@
+This is the sequence diagram for adding a run:
 
-## Non-Functional Requirements
+![Run Sequence Diagram](img/run_sequence_diagram.png)
```
Is the diagram too long? Might affect readability.

LuoYu-uwu commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,662 @@
+
+##### Gym Sequence
+
+![Gym Sequence Diagram](./img/gym_sequence_diagram.jpg)
```
![Screenshot 2024-04-04 123049](https://github.com/nus-cs2113-AY2324S2/tp/assets/110781598/1160dcc1-c214-45ab-ab09-64fae499b480)
Is there inconsistency in naming? The first four says <<class>> and has colon before the name, while the last says <<Class>> and has no colon, resulting in some confusion

LuoYu-uwu commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,662 @@
+6. `HealthList.predictNextPeriodStartDate()` gets the predicted start date by calling `Period.nextCyclePrediction()`.
+7. `Period.getLastThreeCycleLengths()` is called to find the sum of the latest three cycle lengths before returning to `Period.nextCyclePrediction()` for necessary calculation to obtain the predicted start date.
+8. The `Parser` class calls `Period.printNextCyclePrediction()` with the predicted start date as a parameter. This method prints a string indicating the number of days until the predicted start date of the next period, or how many days late the period is if the current date is after the predicted start date.
+![Sequence Diagram for Add Period](img/period_prediction.png)
```
![Screenshot 2024-04-04 124035](https://github.com/nus-cs2113-AY2324S2/tp/assets/110781598/f0534ba8-8470-4129-9aee-e7a9036a8d28)

Is it better to include the parenthesis and/or parameters behind the function name?

TimothyLKM commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,662 @@
+The application follows an Object-Oriented Design approach, with separate classes for handling different components of the application, such as user input, output, exercise logging, and health data management.
+
+
+![ArchitectureDiagram](![img.png](img/architecture_diagram.png)
```
<img width="899" alt="Screenshot 2024-04-04 at 1 20 52 PM" src="https://github.com/nus-cs2113-AY2324S2/tp/assets/110795904/0aa9309d-4727-4d42-bba0-a923d3e61aa2">

Alignment issue on the Architecture diagram.

Leong-ZM commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,662 @@
+    - If the time taken is **25 minutes and 20 seconds**: Use `25:20`.
+    - If the time taken is **1 hour, 20 minutes and 30 seconds**: Use `01:20:30`.
+- `[date]` is in `DD-MM-YYYY` format. **THIS IS AN OPTIONAL PARAMETER**.
 
```
Maybe show sample output?

TimothyLKM commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,662 @@
+
+#### Workout List
+
+{Include `WorkoutList` description}
+
+###### [Back to table of contents](#table-of-contents)
+
+#### Gym
+
+{Include `Gym` description}
+
+###### [Back to table of contents](#table-of-contents)
+
+##### Gym Station
+
+{Include `GymStation` description}
+
+###### [Back to table of contents](#table-of-contents)
+
+##### Gym Set
+
+{Include `GymSet` description}
+
+###### [Back to table of contents](#table-of-contents)
+
+#### Run
+
+{Include `Run` description}
+
```
Missing description.

Leong-ZM commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,662 @@
+###### [Back to table of contents](#table-of-contents)
+
+### Health Package
+![Class Diagram for Health Package](img/health_package.png)
```
probably not necessary to show obvious methods such as getWeight()

TimothyLKM commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,662 @@
+| 2.0     | runner                | quickly view my most recent run details               | see my latest run                     |
+| 2.0     | gym enthusiast        | quickly view my most recent gym session               | see my latest gym session             | 
+| 2.0     | gym enthusiast        | accurately track my progress and strength gains       | enter varying weights for sets        | 
+| 2.0     | female user           | plan ahead and better manage my health                | p#D85D43ict my next period start date | 
```
Typo on the "I want to" column

TimothyLKM commented on docs/img/health_package.png:
Too many unnecessary commands on this Class diagram.

TimothyLKM commented on docs/img/period_prediction.png:
Remove extra headers at the bottom of all your diagrams

TimothyLKM commented on docs/img/period_sequence.png:
Diagram may be too much and cluttered to read. Do consider omitting some commands.

Leong-ZM commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,662 @@
+
+6. The `Appointment` object is passed to `Output.printAddAppointment()` and a message acknowledging the successful adding is printed to the screen.
+
+![AddAppointment](img/appointment_sequence.png)
```
parenthesis should be included in function names

TimothyLKM commented on docs/img/ui_class_diagram.png:
You could omit some less important methods.

TimothyLKM reviewed (commented):
Overall, very detail diagrams but you can omit some less important methods. Good job!

jing-xiang commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,662 @@
+
+The application follows an Object-Oriented Design approach, with separate classes for handling different components of the application, such as user input, output, exercise logging, and health data management.
+
+![Architecture Diagram](img/architecture_diagram.png)
```
formatting error on github pages

![image](https://github.com/nus-cs2113-AY2324S2/tp/assets/61624545/33fbe3d3-1331-4dae-a678-b2c8ac60b190)

jing-xiang commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,662 @@
+
+###### [Back to table of contents](#table-of-contents)
+
+#### Parser
```
![image](https://github.com/nus-cs2113-AY2324S2/tp/assets/61624545/8a482e1a-63bc-4c40-9a93-b7998c6ed0a1)

seems incomplete

jing-xiang commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,662 @@
+
+##### Period Sequence
+
+The sequence diagram below illustrates the process of period prediction.
```
Sequence diagram seems accurate and detailed. Good job.

jing-xiang commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,662 @@
+
+- Station name is `Bench Press`.
+- User has done **2 sets of 3 repetitions**.
+- The first set was done using 100kg, and the second using 200kg.
```
+1 for 200kg bench press 👍

jing-xiang commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,662 @@
+
+###### [Back to table of contents](#table-of-contents)
+
+### View History
```
should these be empty? 

![image](https://github.com/nus-cs2113-AY2324S2/tp/assets/61624545/2a54f2c7-6db4-4aaf-a25b-f24a6e2ce229)

jing-xiang commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,662 @@
+| 2.0     | runner                | quickly view my most recent run details               | see my latest run                     |
+| 2.0     | gym enthusiast        | quickly view my most recent gym session               | see my latest gym session             | 
+| 2.0     | gym enthusiast        | accurately track my progress and strength gains       | enter varying weights for sets        | 
+| 2.0     | female user           | plan ahead and better manage my health                | p#D85D43ict my next period start date | 
```
seems like a formatting/typo error on the female user row?


![image](https://github.com/nus-cs2113-AY2324S2/tp/assets/61624545/86e14977-377a-47a5-a579-cec34225fc51)

jing-xiang commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,662 @@
+</code>
+
+- `[distance]` is a 2 **decimal place positive number** representing the number of kilometers covered.
+- `[time]` is in `HH:MM:SS` or `MM:SS` format with positive integers, representing the amount of time taken for the run.
```
sorry if i missed it but maybe clarify whats the type

![image](https://github.com/nus-cs2113-AY2324S2/tp/assets/61624545/cdbbda49-1095-472d-828d-30a0bcbc38ed)

jing-xiang reviewed (commented):
Overall good DG. Detailed sequence diagrams  but some part seems missing and there are some formatting errors so remember to fix those!

cyhjason29 commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,703 @@
+=======
+
+![ArchitectureDiagram](img/architecture_diagram.png)
+>>>>>>> f712869cb298d4ac1f4d6178e7eaeed520cb1919
```
maybe the image was accidentally repeated when resolving merge conflicts?
![image](https://github.com/nus-cs2113-AY2324S2/tp/assets/110605479/ea358428-1dcb-43f5-8150-76ef44bf1e98)

shawnpong commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,703 @@
+
+#### Workout List
+
+{Include `WorkoutList` description}
```
Description still empty for all these workouts

shawnpong commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,703 @@
+
+#### Filters
+
+{Include `Filters` description}
```
These descriptions are also not filled in yet?

cyhjason29 commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,703 @@
+
+###### [Back to table of contents](#table-of-contents)
+
+#### Health List
```
would class diagrams help to make the following classes clearer?

shawnpong commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,703 @@
+HEALTH /h:bmi /height:[height] /weight:[weight] /date:[date]
+</code>
+
+- `[height]` is a 2 **decimal place positive number** representing the user's height.
+
+- `[weight]`is a 2 **decimal place positive number** representing the user's weight.
+
+- `[date]` is in `DD-MM-YYYY` format (i.e. `19-03-2024`).
```
height, weight and date all have specific inputs that have been defined "decimal place positive number" which is good practice!

cyhjason29 commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,703 @@
+This is the sequence diagram for adding a run:
+
+![Run Sequence Diagram](img/run_sequence_diagram.png)
+
```
should the handleWorkout() arrow be solid instead?
should addRun() be pointing to the top of the activation bar of :WorkoutList instead?
![image](https://github.com/nus-cs2113-AY2324S2/tp/assets/110605479/7ddac46f-8c0d-4497-9487-5c1f31182659)

shawnpong commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,703 @@
+
+## Frequently Asked Questions
+
+1. **Q: How do I set up the development environment for the project?**  
+   A: You can set up the development environment by first cloning the repository to your local system. Then, load the project into your chosen IDE (we recommend IntelliJ IDEA).
+
```
Might want to specify which version of java to use etc?

cyhjason29 commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,703 @@
+| 2.0     | gym enthusiast        | accurately track my progress and strength gains       | enter varying weights for sets        | 
+| 2.0     | female user           | plan ahead and better manage my health                | p#D85D43ict my next period start date | 
+| 2.0     | injured user          | remember the appointments I have                      | track my medical appointments         |
+
```
should the "So that I can" and "I want to" columns be swapped?

rouvinerh committed:
make history consistent

rouvinerh committed:
make latest more consistent

rouvinerh committed:
fix output test case

rouvinerh committed:
add validation of read and write permissions

rouvinerh committed:
add manual testing for dg

j013n3 committed:
Fix error caused by "NA"

rouvinerh committed:
remove ref from dg

syj02 committed:
Merge pull request #225 from rouvinerh/master

write manual testing, add more validation, remove refs in dg

j013n3 committed:
Update DG for addPeriod and addBmi

j013n3 committed:
Edit

rouvinerh committed:
Merge pull request #227 from j013n3/update

Update

Commit:
Modify showEarliestAppointment to showLatestAppointment
*Prints the latest appointment

Commit:
Refactor name of method in Health
showCurrentBmi to showLatestBmi

Commit:
Refactor HealthList
Reorder the methods order

Commit:
Refactor Bmi
Reorder methods order

Commit:
Add HealthList class diagram to DG

Commit:
Refactor to sort order of methods

Commit:
Update health package design in DG

Commit:
Reorder sequence of health implementations and commands

JustinSoh committed:
add PPP and Seq Diagram for History and Latest

rouvinerh committed:
Merge pull request #228 from JustinSoh/master

add PPP and Seq Diagram for History and Latest

Commit:
Update authors tag

Commit:
Reorder the Health sequence

Commit:
Add PPP

rouvinerh committed:
add check before printing appt list after delete

Commit:
Update OutputTest

Commit:
Merge branch 'master' of https://github.com/AY2324S2-T09-4/tp

rouvinerh committed:
add test for more coverage

raajamani committed:
Motivation

raajamani committed:
Motivation 2

raajamani committed:
Merge branch 'master' of https://github.com/raajamani/tp

# Conflicts:
#	docs/DeveloperGuide.md
#	docs/UserGuide.md
#	src/main/java/ui/Output.java

rouvinerh committed:
update period and bmi seq diagrams

L5-Z committed:
Merge pull request #230 from rouvinerh/master

Commit:
Merge branch 'master' of https://github.com/AY2324S2-T09-4/tp

raajamani committed:
Motivate (Updated)

raajamani committed:
Motivation v2

raajamani committed:
M v3

Commit:
Refactor method names, respective codes, and diagrams

raajamani committed:
v4

raajamani committed:
v5

raajamani committed:
v6

raajamani committed:
M + Original UG, DG

Commit:
Remove assertion of endDate cannot be null
Updated to allow input of just start date

JustinSoh committed:
add Integration Testing for show Latest

- covered all printLatest positive test case
- to do negative test cases

JustinSoh committed:
add Integration Testing for show Latest

- covered all printLatest positive test case
- to do negative test cases

JustinSoh committed:
add Integration Testing for show Latest

- covered all printLatest positive test case
- to do negative test cases

Commit:
Modify assertions in period

Commit:
Fix checkstyle errors

rouvinerh committed:
Merge pull request #231 from syj02/master

Update DG and Add PPP

rouvinerh committed:
add delete to dg

JustinSoh committed:
split parseHistoryAndLatest into History and Latest

- change up error handling also

JustinSoh committed:
split parseHistoryAndLatest into History and Latest

- change up error handling also

JustinSoh committed:
split parseHistoryAndLatest into History and Latest

- change up error handling also

rouvinerh committed:
add storage for dg

rouvinerh committed:
replace storage puml with image

rouvinerh committed:
change username to allow numbers

JustinSoh committed:
add more error testing for health

JustinSoh committed:
add more error testing for health

JustinSoh committed:
add more error testing for health

JustinSoh committed:
Merge branch 'master' of https://github.com/JustinSoh/tp

L5-Z committed:
Merge pull request #234 from rouvinerh/master

add delete and storage to dg

JustinSoh committed:
increase coverage for Output

JustinSoh committed:
increase coverage for Output

JustinSoh committed:
Merge branch 'master' of https://github.com/JustinSoh/tp

L5-Z committed:
Merge pull request #235 from JustinSoh/master

integration test + minor exception fixes

raajamani committed:
Motivation (s)

raajamani committed:
M (s) + Original DevGuide

raajamani committed:
jd

raajamani committed:
Mo

j013n3 committed:
Update sequence diagrams for Bmi and Period

j013n3 committed:
Add PPP

Commit:
Update PPP

Commit:
Merge branch 'master' of https://github.com/AY2324S2-CS2113T-T09-4/tp

rouvinerh committed:
Merge pull request #236 from j013n3/dg,ppp

Dg,PPP

rouvinerh committed:
Merge pull request #237 from L5-Z/master

Update PPP

Commit:
Update UG
Change description of latest appointment to be clearer

Commit:
Add missing javadoc in HealthList

Commit:
Add deleteAppointment test for negative index

Commit:
Add deleteBmi test for negative index

Commit:
Add getPeriod test for empty period list

Commit:
Add deletePeriod test for negative index

Commit:
Add predictNextPeriodStartDate test with empty period list

Commit:
Add periodConstructor test for empty end date

Commit:
Add updateEndDate test

Commit:
Add calculatePeriodLength test with null end date

Commit:
Merge branch 'master' of https://github.com/AY2324S2-T09-4/tp

rouvinerh committed:
Merge pull request #238 from syj02/master

Update UG

rouvinerh committed:
add tests for coverage

90% lines!

rouvinerh committed:
remove unneeded method and update class diagram

rouvinerh committed:
fix ppp

rouvinerh committed:
add destroy for sequence diagrams

L5-Z committed:
Merge pull request #239 from rouvinerh/master

add tests, fix ppp

rouvinerh committed:
update diagrams, check dg until add run

rouvinerh committed:
add author tags for markdown

Added since majority of documentation was only paraphrased by me, credits go to the original writers since bulk of work was done by them.

j013n3 committed:
Update sequence diagram and DG

j013n3 committed:
Add puml

j013n3 committed:
Add JUnit tests under validation

j013n3 committed:
Fix missing cycle lengths due to sort

rouvinerh committed:
edit DG diagrams and wording

rouvinerh committed:
Merge pull request #240 from j013n3/validation

Validation

rouvinerh committed:
Merge branch 'master' into master

j013n3 committed:
Merge pull request #241 from rouvinerh/master

finalise DG and add author tags

Commit:
Update: Updated test cases to increase code robustness

JustinSoh committed:
rename WorkoutList to WorkoutLists

JustinSoh committed:
rename WorkoutList to WorkoutLists

JustinSoh committed:
break WorkoutLists.getWorkouts(filter) into smaller methods

- getRun()
- getGym()
- getWorkouts()

JustinSoh committed:
break WorkoutLists.getWorkouts(filter) into smaller methods

- getRun()
- getGym()
- getWorkouts()

JustinSoh committed:
Merge pull request #242 from L5-Z/master

Updated test cases to increase code robustness

JustinSoh committed:
cleanup workoutLists

- applied SLAP to deleteGym and deleteRun
- rearranged methods according to Public/Protected/Private
- touch up JavaDocs

JustinSoh committed:
cleanup workoutLists

- applied SLAP to deleteGym and deleteRun
- rearranged methods according to Public/Protected/Private
- touch up JavaDocs

JustinSoh committed:
Merge branch 'master' of https://github.com/JustinSoh/tp

Commit:
Cleanup Run and Workout

- make it more OOP and abstracted / slap various methods

rouvinerh committed:
add 'back' for gym station, add trims

rouvinerh committed:
add max gym station check

rouvinerh committed:
Merge pull request #243 from JustinSoh/master

cleanup WorkoutLists, Run and Workout

rouvinerh committed:
Merge branch 'master' of https://github.com/rouvinerh/tp

rouvinerh committed:
fix grammar

rouvinerh committed:
refactor run validation to class

Commit:
Update Javadoc

Commit:
Merge branch 'master' of https://github.com/AY2324S2-CS2113T-T09-4/tp

L5-Z committed:
Merge pull request #244 from rouvinerh/master

add gym station exit, refactor run

Commit:
move GymStation validation -> Gym

Commit:
move GymStation validation -> Gym

Commit:
move GymStation validation -> Gym

Commit:
move GymStation validation -> Gym

Commit:
move GymStation validation -> Gym

Commit:
move GymStation validation -> Gym

JustinSoh committed:
Merge branch 'master' of https://github.com/JustinSoh/tp

Commit:
fix javaDoc and cleanup for Gym

Commit:
fix javaDoc and cleanup for Gym

Commit:
fix javaDoc and cleanup for Gym

Commit:
fix javaDoc and cleanup for Gym

Commit:
modify seq diagram

Commit:
modify seq diagram

Commit:
modify seq diagram

Commit:
modify seq diagram

Commit:
fix checkstyle

Commit:
fix checkstyle

Commit:
fix checkstyle

Commit:
fix checkstyle

rouvinerh committed:
rename run methods

rouvinerh committed:
remove unused constants

rouvinerh committed:
refactor constants and magic numbers

rouvinerh committed:
rephrase javadoc, fix filter checking

rouvinerh committed:
update dg

rouvinerh committed:
update dg diagrams

rouvinerh committed:
update run seq diagram

Commit:
Merge branch 'master' of https://github.com/AY2324S2-CS2113T-T09-4/tp

Commit:
Modify UG
* Change minimum number of stations from 0 to 1 under Adding gym stations.
* Modify the description for END_DATE parameter for clarity under Period.
* Add notes under Period.
* Shift note from Prediction to Period.
* Modify description for TIME for clarity under Appointment.
* Fix grammar under History.

Commit:
Update DG
Fix grammar and typos in description:
* under Overview of Components
* below Architecture diagram,
* under UI Package
* under Handler
* under Run class
* under Bmi class
* under Period class
* under Appointment class
* add Bmi
* make Period Prediction
* add Appointment
* under Delete command

Commit:
Modify DG
* Change "right" to "corresponding" for clarity

Commit:
Modify UG
* add "per set" for clarity
* fix typo

Commit:
Update my PPP

j013n3 committed:
edit PPP

j013n3 committed:
Add JavaDocs

j013n3 committed:
Update expected strings

raajamani committed:
Merge branch 'master' of https://github.com/raajamani/tp

# Conflicts:
#	docs/DeveloperGuide.md
#	src/main/java/ui/Handler.java

L5-Z committed:
Merge pull request #247 from syj02/master

Update UG, DG

raajamani committed:
add PPP

raajamani committed:
PPP

rouvinerh committed:
fix logic of adding workout

raajamani committed:
Merge branch 'master' of https://github.com/raajamani/tp

rouvinerh committed:
update run seq

j013n3 committed:
change to line separator.

j013n3 committed:
Merge branch 'master' into healthhh

L5-Z committed:
Merge pull request #248 from j013n3/healthhh

Healthhh

Commit:
touch up sequence diagram

Commit:
touch up sequence diagram

Commit:
touch up sequence diagram

Commit:
touch up sequence diagram

Commit:
Merge branch 'master' of https://github.com/JustinSoh/tp

Commit:
fix english

Commit:
fix english

rouvinerh committed:
Merge branch 'master' into master

JustinSoh committed:
Merge pull request #246 from rouvinerh/master

update javadoc, dg and errors

JustinSoh committed:
Merge branch 'master' of https://github.com/JustinSoh/tp

Commit:
fix merge conflict

raajamani committed:
P

rouvinerh committed:
Merge pull request #245 from JustinSoh/master

clean up Gym

rouvinerh committed:
Merge pull request #251 from raajamani/master

PPP

j013n3 committed:
Change null to NA when deleting period input

j013n3 committed:
shorten parsePeriodInput

L5-Z committed:
Merge pull request #252 from j013n3/refactor

Refactor

Commit:
Fix grammar

Commit:
Update PPP

Commit:
Update Javadocs

Commit:
Merge branch 'master' of https://github.com/AY2324S2-CS2113T-T09-4/tp

rouvinerh committed:
add check on errors and refactor validationtest

rouvinerh committed:
add error checking for runtest

Commit:
small changes

rouvinerh committed:
Merge pull request #253 from L5-Z/master

Update Javadocs and PPP

L5-Z committed:
Merge pull request #255 from rouvinerh/master

update validationtest and runtest

rouvinerh committed:
Merge branch 'master' into master

Commit:
fix checkstyle

Commit:
Merge branch 'master' of https://github.com/JustinSoh/tp

Commit:
fix checkstyle v2

rouvinerh committed:
Merge pull request #254 from JustinSoh/master

update javadoc

rouvinerh committed:
add group code review

JustinSoh committed:
Merge pull request #256 from rouvinerh/master

add group code review

Commit:
update javadocs

rouvinerh committed:
add group edit dg, remove puml

rouvinerh committed:
Merge pull request #257 from JustinSoh/master

update javadocs

L5-Z committed:
Merge pull request #259 from rouvinerh/master

add group edit dg, remove puml

Commit:
update test cases in DG

rouvinerh committed:
Merge pull request #260 from JustinSoh/master

update test cases in DG

j013n3 committed:
update dg

Commit:
Modify period-related images in UG

Commit:
Update PPP

rouvinerh committed:
rearrange method, format dg, edit javadocs

rouvinerh committed:
Merge pull request #261 from syj02/master

Update images in UG

rouvinerh committed:
finalise ppp

rouvinerh committed:
replace author tags for latest printing

j013n3 committed:
Update PPP

L5-Z committed:
Merge pull request #262 from rouvinerh/master

finalise javadoc, move methods, write ppp

rouvinerh committed:
fix test case warnings

Commit:
Auto stash before rebase of "master" onto "AY2324S2-T09-4/master"

Commit:
Update PPP

L5-Z committed:
Merge pull request #264 from JustinSoh/master

Update PPP

rouvinerh committed:
refine dg

rouvinerh committed:
Change multiplicity of gym

Gym can technically have 0 stations.

rouvinerh committed:
add gym station back

j013n3 committed:
remove []

rouvinerh committed:
Merge pull request #263 from j013n3/dggg

Dggg

rouvinerh committed:
Merge branch 'master' into master

Commit:
Update PPP

Commit:
Update PPP

Commit:
Merge branch 'master' of https://github.com/JustinSoh/tp

L5-Z committed:
Merge pull request #265 from rouvinerh/master

update dg diagrams and clean up warnings

rouvinerh committed:
add assertions

rouvinerh committed:
Merge branch 'master' of https://github.com/rouvinerh/tp

rouvinerh committed:
update ug

rouvinerh committed:
add horizontal line

L5-Z committed:
Merge pull request #266 from rouvinerh/master

add assertions for run, group edit ug

rouvinerh committed:
Merge pull request #267 from JustinSoh/master

Update PPP final

rouvinerh committed:
update table

rouvinerh committed:
remove numbering

rouvinerh committed:
edit ppp

Commit:
Update PPP

Commit:
Merge branch 'master' of https://github.com/AY2324S2-CS2113T-T09-4/tp

L5-Z committed:
Merge pull request #268 from rouvinerh/master

update ug table

rouvinerh committed:
Merge pull request #269 from L5-Z/master

Update PPP

rouvinerh committed:
add page breaks

JustinSoh committed:
Merge pull request #271 from rouvinerh/master

add page breaks

rouvinerh committed:
fix pdf formats

JustinSoh committed:
Merge pull request #273 from rouvinerh/master

fix pdf formats

rouvinerh committed:
fix ghost pages

rouvinerh committed:
remove page breaks

JustinSoh committed:
Merge pull request #274 from rouvinerh/master

fix ghost pages

rouvinerh committed:
remove unneeded page

L5-Z committed:
Merge pull request #275 from rouvinerh/master

finalise DG and UG with breaks
